tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/method/FirebaseSubscriptionGetMethod.scala

```diff
@@ -0,0 +1,67 @@
+package com.linagora.tmail.james.jmap.method
+
+import com.google.inject.AbstractModule
+import com.google.inject.multibindings.Multibinder
+import com.linagora.tmail.james.jmap.firebase.FirebaseSubscriptionRepository
+import com.linagora.tmail.james.jmap.json.FirebaseSubscriptionSerializer
+import com.linagora.tmail.james.jmap.method.CapabilityIdentifier.LINAGORA_FIREBASE
+import com.linagora.tmail.james.jmap.model.{FirebaseSubscription, FirebaseSubscriptionGetRequest, FirebaseSubscriptionGetResponse, FirebaseSubscriptionId, FirebaseSubscriptionIds}
+import eu.timepit.refined.auto._
+import org.apache.james.core.Username
+import org.apache.james.jmap.core.CapabilityIdentifier.{CapabilityIdentifier, JMAP_CORE}
+import org.apache.james.jmap.core.Invocation
+import org.apache.james.jmap.core.Invocation.{Arguments, MethodName}
+import org.apache.james.jmap.json.ResponseSerializer
+import org.apache.james.jmap.method.{InvocationWithContext, Method, MethodWithoutAccountId}
+import org.apache.james.jmap.routes.SessionSupplier
+import org.apache.james.mailbox.MailboxSession
+import org.apache.james.metrics.api.MetricFactory
+import org.reactivestreams.Publisher
+import play.api.libs.json.{JsError, JsObject}
+import reactor.core.scala.publisher.{SFlux, SMono}
+
+import scala.jdk.CollectionConverters._
+
+class FirebaseSubscriptionGetMethodModule extends AbstractModule {
+  override def configure(): Unit = {
+    Multibinder.newSetBinder(binder(), classOf[Method])
+      .addBinding()
+      .to(classOf[FirebaseSubscriptionGetMethod])
+  }
+}
+
+class FirebaseSubscriptionGetMethod()(val repository: FirebaseSubscriptionRepository,
+                                      val metricFactory: MetricFactory,
+                                      val sessionSupplier: SessionSupplier) extends MethodWithoutAccountId[FirebaseSubscriptionGetRequest] {
```

Comment:
No @Inject ?

Reply:
I created it manually without annotation, (because we have a enable/disable firebase feature)

Reply:
Where?

Just add the module inside the module chooser and rely on standard injects please.